src/docs_main.cr

```diff
@@ -7,6 +7,7 @@ require "./big"
 require "./compiler/crystal/macros"
 require "./compress/**"
 require "./crypto/**"
+require "./crystal/syntax_highlighter/*"
```

Comment:
Note that this require in turn has `require "compiler/crystal/syntax/**"` which may in the future add undesirable types into the docs output.

**It already adds a lot of undesirable types into the docs output.**

<details>
<summary>
Crystal::
</summary>
Alias And AnnotationDef Annotation Arg ArrayLiteral Asm AsmOperand Assign ASTNode BinaryOp Block BoolLiteral Break Call Case Cast CharLiteral ClassDef ClassVar CodeError ControlExpression CStructOrUnionDef Def DoubleSplat EnumDef ErrorFormat Error ExceptionHandler Expressions Extend ExternalVar FunDef Generic Generic Global HashLiteral HashLiteral If ImplicitObj Include InstanceSizeOf InstanceVar IsA Lexer Lexer LibDef Location MacroExpression MacroFor Macro MacroIf MacroLiteral MacroVar MacroVerbatim MagicConstant Metaclass ModuleDef MultiAssign NamedArgument NamedTupleLiteral NamedTupleLiteral Next NilableCast NilLiteral Nop Not NumberLiteral OffsetOf OpAssign Or Out Parser Parser Path PointerOf ProcLiteral ProcNotation ProcPointer RangeLiteral ReadInstanceVar RegexLiteral Require Rescue RespondsTo Return Select Select Self SizeOf SpecialVar Splat StringInterpolation StringLiteral SymbolLiteral SyntaxException SyntaxHighlighter SyntaxHighlighter Token Token ToSVisitor Transformer TupleLiteral TypeDeclaration TypeDef TypeOf UnaryExpression Underscore UninitializedVar Union Unless Until Var VirtualFile Visibility VisibilityModifier Visitor When While Yield
</details>

Which also increases the duration of `make docs` by 20% (for me 8.50s -> 10.25s)

Reply:
Ah, that's a downside of requiring compiler code from stdlib 😞

Reply:
I resolved this by removing the require of compiler code when building docs.